src/components/swap/SwapPlatformSelector.tsx

```diff
@@ -1,31 +1,38 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useContext, useEffect, useState } from 'react'
 import { CurrencyAmount, Percent, Trade, RoutablePlatform, TradeType, UniswapV2Trade } from '@swapr/sdk'
 import { AutoColumn } from '../Column'
 import { TYPE } from '../../theme'
 import CurrencyLogo from '../CurrencyLogo'
-import { Box, Flex } from 'rebass'
-import Radio from '../Radio'
-import QuestionHelper from '../QuestionHelper'
+import { Box, Flex } from 'rebass/styled-components'
 import WarningHelper from '../WarningHelper'
 import SwapRoute from './SwapRoute'
 import { useSwapsGasEstimations } from '../../hooks/useSwapsGasEstimate'
 import { useUserSlippageTolerance } from '../../state/user/hooks'
 import { useSwapState } from '../../state/swap/hooks'
 import { useGasFeesUSD } from '../../hooks/useGasFeesUSD'
-import { RowFixed } from '../Row'
-import { ONE_BIPS, PRICE_IMPACT_MEDIUM, ROUTABLE_PLATFORM_LOGO } from '../../constants'
-import styled from 'styled-components'
-import { computeSlippageAdjustedAmounts, computeTradePriceBreakdown, simpleWarningSeverity } from '../../utils/prices'
+import {
+  computeSlippageAdjustedAmounts,
+  computeTradePriceBreakdown,
+  limitNumberDecimalPlaces,
```

Comment:
Not sure though.

```suggestion
  limitNumberOfDecimalPlaces,
```

Reply:
I was thinking about "limit number's decimal places" but it also could be "limit decimal places" or "limit decimal places number". I liking "limitDecimalPlaces" but don't know if it's to vague

Reply:
Implemented your suggestion.